readme.md

```diff
@@ -162,13 +213,13 @@ Your Rails app has the following `application.html.erb`. Nothing shows up in you
 </html>
 ```
 
-> Your answer...
+To start with I would close the <!DOCTYPE html tag with a corresponding ">".  Second, I would change the name to layout.erb.  That should fix the problem.  If not I would have to do some document dives to see if the stylesheet_link_tag, javascript_include_tag, and csrf_meta_tags helper methods within the clown hat tags are setup properly.
 
```

Comment:
Like your reasoning! In Rails, we should be find with the default `application.html.erb` ,  but the more subtle thing is that we need an `=` in the `<%` before `yield` as we want to make sure to print everything inside the block to the screen!